modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class Client
+    ##
+    # HTTP POST call to the VEIS Auth endpoint to get the access token
+    #
+    # @return [Faraday::Response]
+    #
+    def request_veis_token
+      auth_url = Settings.travel_pay.veis.auth_url
+      tenant_id = Settings.travel_pay.veis.tenant_id
```

Comment:
should just be `Settings.travel_pay.auth_url` or `Settings.travel_pay.tenant_id`

Reply:
No, the settings here (`auth_url` and `tenant_id`) only apply to the VEIS endpoints. The BTSSS has no concept of an auth url or a tenant.

The way the API is created, you must first authorize with VEIS before you can call additional endpoints.

Reply:
Ah I see what you were getting at, I think the comment below cleared things up for me. You're saying in the `settings.yml`, the values are `travel_pay.auth_url` and `travel_pay.tenant_id` currently. I think a quick face-to-face chat will clear this up.